tests/aws/services/apigateway/test_apigateway_import.py

```diff
@@ -874,6 +879,13 @@ def test_import_with_cognito_auth_identity_source(
 
         rest_api_id = response["id"]
 
-        # assert that are no multiple authorizers
         authorizers = aws_client.apigateway.get_authorizers(restApiId=rest_api_id)
-        snapshot.match("get-authorizers", authorizers)
+        snapshot.match("get-authorizers", sorted(authorizers["items"], key=lambda x: x["name"]))
```

Comment:
very minor nit: could also use `itemgetter("name")` here like the `sorted` call below.

Reply:
Honestly, don't have to change it, the CI is green, let's keep it!